tests/Integration/Behaviour/Features/Scenario/Module/module.feature

```diff
@@ -43,6 +43,24 @@ Feature: Module
     Given module ps_featuredproducts has following infos:
       | enabled | true |
 
+  Scenario: Reset module status
+    Given module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts |
+      | enabled        | true |
+      | installed      | true |
+    When I reset module "ps_featuredproducts"
+    Given module ps_featuredproducts has following infos:
+      | technical_name | ps_featuredproducts |
+      | enabled        | true |
+      | installed      | false |
+    Then I should have an exception that module is not installed
```

Comment:
I believe here you are testing that an exception is correctly thrown when you try to reset a module that is not installed or enabled, but this read step won't change the status into what you expect you need to use a write/update action to do that

 For now, we don't have a step to uninstall but we have one to disable
```suggestion
    When I disable module "ps_featuredproducts"
    And I reset module "ps_featuredproducts"
    Then I should have an exception that module is not enabled
```